src/get-env-variables.js

```diff
@@ -0,0 +1,60 @@
+const exec = require('child_process').exec
+// import spawn from 'spawn-command-with-kill'
```

Comment:
Okay, another 2 hours of trying to get it to work and I have been left frustrated...

When I run inside of this repo:
let child = spawn('npm run env')
child.stdout.on('data', () => {})
// throws error can't read on of undefined

When I run outside of this repo, everything works just fine...

If you have time to help that would be greatly appreciated, otherwise I may not be any use for moving forward on this :(